apps/complaints/index.js

```diff
@@ -1,16 +1,27 @@
 'use strict';
 
+const controllers = require('hof').controllers;
+
 module.exports = {
 	name: 'complaints',
 	steps: {
-		'/': {
-			template: 'step',
-			fields: ['name-text'],
-			next: '/step1'
-		},
-		'/step1': {
-			template: 'step',
-			fields: ['name-text']
+    '/': {
+      controller: controllers.start,
+      next: '/who'
+    },
+		'/who': {
+			fields: ['who-radio'],
+			next: '/fullname',
```

Comment:
as we will have 2 name steps (potentially) can we call them 'applicant-name' and 'representative-name'